### PR TITLE
Update API version, improvements in API controller.

### PR DESCRIPTION
**Pull request by kegor**
**Problem:**
We would like to update version of the Editor API to make possible to use all new features added in the newest versions.

**Solution:**
Currently were made initial changes only in the EditorApiController.
These changes were tested along with https://github.com/groupdocs-total/GroupDocs.Total-Angular/pull/168.

**Screenshots:**
You could take a look at the screenshots in the following related PRs:
- https://github.com/groupdocs-total/GroupDocs.Total-Angular/pull/166
- https://github.com/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pull/8

### CODE REVIEW

File: src/Products/Editor/Controllers/EditorApiController.cs [hunk 1]
```diff
@@ -355,106 +357,142 @@ private dynamic GetSaveFormat(string saveFilePath)
...
                     format = WordProcessingFormats.Docx;
                     break;
-
             }
+
             return format;
         }
 
-        private dynamic GetSaveOptions(string saveFilePath)
+        private ISaveOptions GetSaveOptions(string saveFilePath)
```
Comment (bobkovalex) [inline src/Products/Editor/Controllers/EditorApiController.cs]: Codacy Issue found: [Remove the unused private method 'GetSaveOptions'.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074493)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 3
           

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -5
         

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074493&bid=16695619)

File: src/Products/Editor/Controllers/EditorApiController.cs [hunk 2]
```diff
@@ -355,106 +357,142 @@ private dynamic GetSaveFormat(string saveFilePath)
...
             if (options == null)
             {
-                options = new SpreadsheetSaveOptions();
+                options = new SpreadsheetSaveOptions(SpreadsheetFormats.Xlsb);
             }
+
+            return options;
+        }
+
+        private ILoadOptions GetLoadOptions(string guid)
```
Comment (bobkovalex) [inline src/Products/Editor/Controllers/EditorApiController.cs]: Codacy Issue found: [Make 'GetLoadOptions' a static method.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074493)

File: src/Products/Editor/Controllers/EditorApiController.cs (hunk 1 above)
Comment (bobkovalex) [inline src/Products/Editor/Controllers/EditorApiController.cs]: Codacy Issue found: [Make 'GetSaveOptions' a static method.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074493)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -5
         

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074570&bid=16695825)

File: src/Products/Editor/Controllers/EditorApiController.cs (hunk 1 above)
Comment (egorovpavel) [inline src/Products/Editor/Controllers/EditorApiController.cs]: Codacy Issue found: [Remove the unused private method 'GetSaveOptions'.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074570)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -17
         

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074493&bid=16695619)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -17
         

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074570&bid=16695825)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -17
         

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074570&bid=16695825)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -17
         

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074493&bid=16695619)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -17
         

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074493&bid=16695619)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -17
         

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074570&bid=16695825)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -17
         

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074493&bid=16695619)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -17
         

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074570&bid=16695825)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074493&bid=16695619)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5074570&bid=16695825)

**Pull request merged**